### ISSUE TITLE
Please follow the XDG base directory specification

### ISSUE THREAD

**Issue by sorsasampo** [ProtonVPN/protonvpn-cli#159]
Currently protonvpn-cli writes a mish-mash of conf, log and cache files in `~/.protonvpn-cli/`.

This is problematic for many reasons, some examples being 1) having `~` under version control and 2) backups. Excluding cache & log files is much easier if they are in their own places.

Please follow the [XDG base directory specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html).

**Comment by jeffshee** [comment]
Other than that, please make the whole directories and the files inside accessible without root as well. Currently access to `~/.protonvpn-cli/` requires root every time, which is inconvenient for the log files.

**Comment by jeffshee** [comment]
@sorsasampo yeah that sounds pretty a common sense to me. Normal files, especially log files should be accessible to normal user. If you want to write root files, then write it to somewhere else that nobody will complaint about.
